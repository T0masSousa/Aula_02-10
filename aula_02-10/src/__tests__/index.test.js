import React from 'react';
import { render } from '@testing-library/react';
import Home from '../pages/index';

test('renderiza corretamente', () => {
  const { asFragment } = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});