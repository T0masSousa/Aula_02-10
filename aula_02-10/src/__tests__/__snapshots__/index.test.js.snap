// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Index 1`] = `
<div>
  <main>
    <img
      alt="Next.js logo"
      data-nimg="1"
      decoding="async"
      fetchpriority="high"
      height={38}
      onError={[Function]}
      onLoad={[Function]}
      src="https://nextjs.org/icons/next.svg"
      style={
        {
          "color": "transparent",
        }
      }
      width={180}
    />
    <ol>
      <li>
        Get started by editing 
        <code>
          src/pages/index.js
        </code>
        .
      </li>
      <li>
        Save and see your changes instantly.
      </li>
    </ol>
    <div>
      <a
        href="https://vercel.com/new?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Vercel logomark"
          data-nimg="1"
          decoding="async"
          height={20}
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          src="https://nextjs.org/icons/vercel.svg"
          style={
            {
              "color": "transparent",
            }
          }
          width={20}
        />
        Deploy now
      </a>
      <a
        href="https://nextjs.org/docs?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        Read our docs
      </a>
    </div>
  </main>
  <footer>
    <a
      href="https://nextjs.org/learn?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="File icon"
        aria-hidden={true}
        data-nimg="1"
        decoding="async"
        height={16}
        loading="lazy"
        onError={[Function]}
        onLoad={[Function]}
        src="https://nextjs.org/icons/file.svg"
        style={
          {
            "color": "transparent",
          }
        }
        width={16}
      />
      Learn
    </a>
    <a
      href="https://vercel.com/templates?framework=next.js&utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Window icon"
        aria-hidden={true}
        data-nimg="1"
        decoding="async"
        height={16}
        loading="lazy"
        onError={[Function]}
        onLoad={[Function]}
        src="https://nextjs.org/icons/window.svg"
        style={
          {
            "color": "transparent",
          }
        }
        width={16}
      />
      Examples
    </a>
    <a
      href="https://nextjs.org?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Globe icon"
        aria-hidden={true}
        data-nimg="1"
        decoding="async"
        height={16}
        loading="lazy"
        onError={[Function]}
        onLoad={[Function]}
        src="https://nextjs.org/icons/globe.svg"
        style={
          {
            "color": "transparent",
          }
        }
        width={16}
      />
      Go to nextjs.org →
    </a>
  </footer>
</div>
`;

exports[`renderiza corretamente 1`] = `
<DocumentFragment>
  <div>
    <main>
      <img
        alt="Next.js logo"
        data-nimg="1"
        decoding="async"
        fetchpriority="high"
        height="38"
        src="https://nextjs.org/icons/next.svg"
        style="color: transparent;"
        width="180"
      />
      <ol>
        <li>
          Get started by editing 
          <code>
            src/pages/index.js
          </code>
          .
        </li>
        <li>
          Save and see your changes instantly.
        </li>
        <p>
          AAAAAAA
        </p>
      </ol>
      <div>
        <a
          href="https://vercel.com/new?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="Vercel logomark"
            data-nimg="1"
            decoding="async"
            height="20"
            loading="lazy"
            src="https://nextjs.org/icons/vercel.svg"
            style="color: transparent;"
            width="20"
          />
          Deploy now
        </a>
        <a
          href="https://nextjs.org/docs?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
          rel="noopener noreferrer"
          target="_blank"
        >
          Read our docs
        </a>
      </div>
    </main>
    <footer>
      <a
        href="https://nextjs.org/learn?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="File icon"
          aria-hidden="true"
          data-nimg="1"
          decoding="async"
          height="16"
          loading="lazy"
          src="https://nextjs.org/icons/file.svg"
          style="color: transparent;"
          width="16"
        />
        Learn
      </a>
      <a
        href="https://vercel.com/templates?framework=next.js&utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Window icon"
          aria-hidden="true"
          data-nimg="1"
          decoding="async"
          height="16"
          loading="lazy"
          src="https://nextjs.org/icons/window.svg"
          style="color: transparent;"
          width="16"
        />
        Examples
      </a>
      <a
        href="https://nextjs.org?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Globe icon"
          aria-hidden="true"
          data-nimg="1"
          decoding="async"
          height="16"
          loading="lazy"
          src="https://nextjs.org/icons/globe.svg"
          style="color: transparent;"
          width="16"
        />
        Go to nextjs.org →
      </a>
    </footer>
  </div>
</DocumentFragment>
`;
